src/common/components/exchange/TxSummaryFixed.vue

```diff
@@ -746,6 +746,10 @@ export default defineComponent({
       window.open(`${appConstants.DERIVE_BTC_ADDRESS_DOCUMENTATION_URL}`);
     }
 
+    function cropHash(hash: string):string {
+      return `${hash.substr(0, 14)}...${hash.substr(hash.length - 14, hash.length)}`;
```

Comment:
I know you might have seen it used across the app but `substr` is deprecated. We should use `slice` or `substring` instead.

Reply:
Good catch, I'ved change it [here](https://github.com/rsksmart/2wp-app/pull/721/commits/5edf1c9e7e0f12f44c670b074ba04e87663ce84c) since this PR depends on [this](https://github.com/rsksmart/2wp-app/pull/721) one to be merged.